src/Compilers/VisualBasic/Portable/Symbols/Source/SourceAssemblySymbol.vb

```diff
@@ -1080,6 +1092,19 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 ValidateIntegralAttributeNonNegativeArguments(attrData, arguments.AttributeSyntaxOpt, diagnostics)
             ElseIf attrData.IsTargetAttribute(Me, AttributeDescription.GuidAttribute) Then
                 attrData.DecodeGuidAttribute(arguments.AttributeSyntaxOpt, diagnostics)
+
+                Dim guidString As String = Nothing
+                If attrData.TryGetGuidAttributeValue(guidString) Then
```

Comment:
>TryGetGuidAttributeValue [](http://example.com/codeflow?start=28&length=24)

Perhaps should simply return string from ```DecodeGuidAttribute``` call above. And we might want to align its implementation with C#.